### ISSUE TITLE
Subtitle popup should be keyboard navigatable

### ISSUE THREAD

**Issue by ttepasse** [lhc70000/iina#1617]
**System and IINA version:**

- macOS 10.13.4
- IINA 1.0.0-beta1

**Expected behavior:**

I use the Search Online for Subtitles functionality. A popup opens, two subtitles found. Ideally I'd like to press Enter to download just one of them. I believe that was standard functionality in 0.0.15

**Actual behavior:**

Enter does nothing. I tried selecting with the Arrow Keys. Nothing happens. I select one with the mouse like an barbarian. Then I type Enter. Nothing happens. So I click on Download with the mouse.
